worker/pkg/benthos/json/sql_processor.go

```diff
@@ -0,0 +1,77 @@
+package neosync_benthos_json
+
+import (
+	"context"
+	"time"
+
+	"github.com/warpstreamlabs/bento/public/service"
+)
+
+func sqlToJsonProcessorConfig() *service.ConfigSpec {
+	return service.NewConfigSpec()
+}
+
+func RegisterSqlToJsonProcessor(env *service.Environment) error {
+	return env.RegisterBatchProcessor(
+		"sql_to_json",
```

Comment:
Is this mysql specific or sql-generic specifc ? or is the intention to eventually support all 3 within this processor?

Reply:
Not entirely sure yet. For now keeping it sql generic until I get a better idea of how much needs to be fixed for postgres and mssql. if the logic get complicated then I plan to split them into driver specific processors